<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stocks App</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(31, 34, 41);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>-11</x>
      <y>-30</y>
      <width>171</width>
      <height>701</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QGroupBox" name="menuGroupBox">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(51, 54, 61);border-color:rgb(51, 54, 61);
border:none;</string>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
       <widget class="QPushButton" name="aboutButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>440</y>
          <width>121</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 900 10pt &quot;Arial Black&quot;;
border-radius:10px;
background-color:rgb(7, 197, 216);</string>
        </property>
        <property name="text">
         <string>About</string>
        </property>
       </widget>
       <widget class="QPushButton" name="back">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>550</y>
          <width>121</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 900 10pt &quot;Arial Black&quot;;
border-radius:10px;
background-color:rgb(7, 197, 216);</string>
        </property>
        <property name="text">
         <string>Home</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="Line" name="line">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(7, 197, 216);</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(7, 197, 216);</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_6">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(7, 197, 216);</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="mainChartButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>220</y>
          <width>121</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 900 10pt &quot;Arial Black&quot;;
border-radius:10px;
</string>
        </property>
        <property name="text">
         <string>Main chart</string>
        </property>
       </widget>
       <widget class="QPushButton" name="otherChartButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>280</y>
          <width>131</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 900 10pt &quot;Arial Black&quot;;
border-radius:10px;
</string>
        </property>
        <property name="text">
         <string>Dividend yield</string>
        </property>
       </widget>
       <widget class="QPushButton" name="summaryButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>121</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 900 10pt &quot;Arial Black&quot;;
border-radius:10px;
</string>
        </property>
        <property name="text">
         <string>Summary</string>
        </property>
       </widget>
       <widget class="QPushButton" name="PSChartButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>340</y>
          <width>131</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 900 10pt &quot;Arial Black&quot;;
border-radius:10px;
</string>
        </property>
        <property name="text">
         <string>P/S</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>-30</y>
      <width>901</width>
      <height>701</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QGroupBox" name="actionGroupBox">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(31, 34, 41);
border:none;</string>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>180</y>
          <width>311</width>
          <height>191</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="firstAddVerticalLayout"/>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>380</y>
          <width>311</width>
          <height>221</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="secondAddVerticalLayout"/>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>180</y>
          <width>411</width>
          <height>421</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="mainChartVerticalLayout"/>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>50</y>
          <width>131</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(70, 224, 130);
color: rgb(255, 255, 255);
font: 900 12pt &quot;Arial Black&quot;;
border-radius:20px;
</string>
        </property>
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>60</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(51, 54, 61);
font: 900 9pt &quot;Arial Black&quot;;
border-radius:3px;
</string>
        </property>
        <property name="maximum">
         <number>400</number>
        </property>
        <property name="value">
         <number>230</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_2">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>60</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(51, 54, 61);
font: 900 9pt &quot;Arial Black&quot;;
border-radius:3px;
</string>
        </property>
        <property name="maximum">
         <number>400</number>
        </property>
        <property name="value">
         <number>232</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_3">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>60</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(51, 54, 61);
font: 900 9pt &quot;Arial Black&quot;;
border-radius:3px;
</string>
        </property>
        <property name="maximum">
         <number>400</number>
        </property>
        <property name="value">
         <number>234</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_4">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>60</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(51, 54, 61);
font: 900 9pt &quot;Arial Black&quot;;
border-radius:3px;
</string>
        </property>
        <property name="maximum">
         <number>400</number>
        </property>
        <property name="value">
         <number>236</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_5">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>60</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(51, 54, 61);
font: 900 9pt &quot;Arial Black&quot;;
border-radius:3px;
</string>
        </property>
        <property name="maximum">
         <number>400</number>
        </property>
        <property name="value">
         <number>235</number>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
